src/scripts/core/contour.js

```diff
@@ -343,6 +343,45 @@
             return this;
         },
 
+        /**
+        * Clears this Contour instance and all its visualizations of any size information so that on the next call to render the instace is re-measured.
+        * 
+        * The function takes two optional arguements width, height -- if given a specific width/height the chart will use that sizing information on the next render.
+        * ### Example:
+        *
+        *     var contour = new Contour({ el:'.myChart' })
+        *           .pie([1,2,3])
+        *           .render()
+        *     
+        *     var onResize = function(e) {
+        *          contour.resize().render();
+        *     }
+        *
+        *     window.addEventListener('resize', onResize);
+        *
+        * @function resize
+        *
+        */
+        resize: function(width, height) {
```

Comment:
I think this approach works for most of the cases, can you add a @param entry for the parameters in the comments?